.github/workflows/reusable-api-e2e.yml

```diff
@@ -90,6 +90,7 @@ jobs:
         run: CI='' pnpm build:api --skip-nx-cache
 
       - name: Run E2E tests
+        if: ${{ !inputs.ee }}
```

Comment:
don't run these tests if we are only interested in the ee tests

Reply:
Nice! Been meaning to address this one 🙏